scripts/custom_checks/version.py

```diff
@@ -23,7 +23,7 @@ def pyproj_version_matches() -> Optional[str]:
 
 
 def _parse_changelog() -> Iterator[Match[str]]:
-    changelog = (CWD / "CHANGELOG.md").read_text()
+    changelog = (CWD / "CHANGELOG.md").read_text(encoding="utf-8")
```

Comment:
😂 (sorry lol)

Reply:
No worries, an edge case which is not picked up by the windows testing.